pkgs/development/libraries/at-spi2-core/default.nix

```
@@ -30,7 +30,8 @@ stdenv.mkDerivation rec {
   outputs = [ "out" "dev" ];
 
   nativeBuildInputs = [ meson ninja pkgconfig gobject-introspection makeWrapper ];
-  buildInputs = [ dbus glib libX11 libXtst libXi ];
+  buildInputs = [ libX11 libXtst libXi ];
+  propagatedBuildInputs = [ dbus glib ];
```

```suggestion
  # in atspi-2.pc dbus-1 glib-2.0
  propagatedBuildInputs = [ dbus glib ];
```

I like to comment that so I can keep an eye on the pc files.
(really would like to figure out why we patch pkgconfig someday)